File: BackEnd/ServicioEventos/Modelo/Invitacion.cs
using System;
using System.Collections.Generic;

namespace ServicioEventos.Modelo;

public partial class Invitacion
{
    public int IdInvitacion { get; set; }

    public int? IdEvento { get; set; }

    public string? Nombre { get; set; }

    public int? Adultos { get; set; }

    public int? Menores { get; set; }

    public DateTime? FechaExpiracion { get; set; }

    public string? Estado { get; set; }

    public string? Telefono { get; set; }

    public virtual Evento? IdEventoNavigation { get; set; }

    public virtual InvitacionConfirmacion? InvitacionConfirmacion { get; set; }
}

File: FrontEnd/src/Modelos/invitacion.ts
export interface Invitacion {
    idInvitacion: number;
    idEvento: number;
    nombre: string;
    adultos: number;
    menores: number;
    fechaExpiracion: Date;
    estado: string;
    invitacionConfirmacion: invitacionConfirmacion|null;
    telefono:string;
  }
  
  export interface invitacionConfirmacion{
    adultosConfirmados: number;
    menoresConfirmados: number;
    idInvitacion: number;
  }

  export interface Evento{
    correo: string;
    mensajeInvitacion: string;
    anfitrion: string;
    idEvento: number;
    fecha: string;
  }